src/wmtk/TetMeshOperationExecutor.cpp

```diff
@@ -700,59 +665,9 @@ Tuple TetMesh::TetMeshOperationExecutor::collapse_edge()
     auto incident_tets_and_faces = get_incident_tets_and_faces(m_operating_tuple);
```

Comment:
You should change the return type of `get_incident_tets_and_faces` to a `std::tuple`. Then you can use `const auto [tets, faces] = get_incident_tets_and_faces()` here.